src/EA.Weee.Web/Areas/Admin/Views/Producers/_detailsResults.cshtml

```diff
@@ -10,581 +10,575 @@
-                    <td class="govuk-table__cell numeric">@scheme.ProducerEeeDetails.DisplayTonnage(QuarterSelection.Q4, WeeeCategory.AutomaticDispensers, ObligationType.B2B)</td>
-                </tr>
-                <tr class="govuk-table__row">
-                    <td class="govuk-table__cell">@((int)WeeeCategory.DisplayEquipment) - @(WeeeCategory.DisplayEquipment.ToDisplayString())</td>
-                    <td class="govuk-table__cell numeric">@scheme.ProducerEeeDetails.DisplayTonnage(QuarterSelection.Q4, WeeeCategory.DisplayEquipment, ObligationType.B2C)</td>
-                    <td class="govuk-table__cell numeric">@scheme.ProducerEeeDetails.DisplayTonnage(QuarterSelection.Q4, WeeeCategory.DisplayEquipment, ObligationType.B2B)</td>
-                </tr>
-                <tr class="govuk-table__row">
-                    <td class="govuk-table__cell">@((int)WeeeCategory.CoolingApplicancesContainingRefrigerants) - @(WeeeCategory.CoolingApplicancesContainingRefrigerants.ToDisplayString())</td>
-                    <td class="govuk-table__cell numeric">@scheme.ProducerEeeDetails.DisplayTonnage(QuarterSelection.Q4, WeeeCategory.CoolingApplicancesContainingRefrigerants, ObligationType.B2C)</td>
-                    <td class="govuk-table__cell numeric">@scheme.ProducerEeeDetails.DisplayTonnage(QuarterSelection.Q4, WeeeCategory.CoolingApplicancesContainingRefrigerants, ObligationType.B2B)</td>
-                </tr>
-                <tr class="govuk-table__row">
-                    <td class="govuk-table__cell">@((int)WeeeCategory.GasDischargeLampsAndLedLightSources) - @(WeeeCategory.GasDischargeLampsAndLedLightSources.ToDisplayString())</td>
-                    <td class="govuk-table__cell numeric">@scheme.ProducerEeeDetails.DisplayTonnage(QuarterSelection.Q4, WeeeCategory.GasDischargeLampsAndLedLightSources, ObligationType.B2C)</td>
-                    <td class="govuk-table__cell numeric">@scheme.ProducerEeeDetails.DisplayTonnage(QuarterSelection.Q4, WeeeCategory.GasDischargeLampsAndLedLightSources, ObligationType.B2B)</td>
-                </tr>
-                <tr class="govuk-table__row">
-                    <td class="govuk-table__cell">@((int)WeeeCategory.PhotovoltaicPanels) - @(WeeeCategory.PhotovoltaicPanels.ToDisplayString())</td>
-                    <td class="govuk-table__cell numeric">@scheme.ProducerEeeDetails.DisplayTonnage(QuarterSelection.Q4, WeeeCategory.PhotovoltaicPanels, ObligationType.B2C)</td>
-                    <td class="govuk-table__cell numeric">@scheme.ProducerEeeDetails.DisplayTonnage(QuarterSelection.Q4, WeeeCategory.PhotovoltaicPanels, ObligationType.B2B)</td>
-                </tr>
-            </tbody>
-        </table>
+        <div class="govuk-tabs__panel" id="registration-details">
+            <h1 class="sr-only" tabindex="0">tab 1 content Overview</h1>
+            <header>
+                <h1 class="govuk-heading-l" aria-hidden="true" style="margin-top: 10px">
```

Comment:
I dont think we need the heading for this one as the tab previously have the heading. I think we should be ok just with the hidden heading